flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveInputFormat.java

```diff
@@ -132,10 +132,10 @@ private RowType tableRowType() {
         return RowType.of(types, fieldNames);
     }
 
-    private BulkFormat<RowData, ? super HiveSourceSplit> createBulkFormatForSplit(
+    public BulkFormat<RowData, ? super HiveSourceSplit> createBulkFormatForSplit(
```

Comment:
why the access level should be changed ?

Reply:
Mistake!